### PR TITLE
Scaffold models from database

### PR DESCRIPTION
**Pull request by kuzmiich**
Pull changes from https://github.com/aces47/DIMS-Core.Double-7
Scaffold models

### CODE REVIEW

File: DIMS-Core/DIMS-Core.BusinessLayer/Converters/DirectionConverter.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class DirectionConverter : JsonConverter<int>
         {
             var directionName = reader.GetString();
 
-            using var context = new DimsCoreContext();
+            using var context = new DIMSCoreContext();
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.BusinessLayer/Converters/DirectionConverter.cs:27]: return back name of context

File: DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj [hunk 2]
```diff
@@ -27,4 +27,8 @@
     <ProjectReference Include="..\DIMS-Core.Common\DIMS-Core.Common.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Models" />
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj:31]: Why do you need this folder here?

File: DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs [hunk 3]
```diff
@@ -1,119 +1,244 @@
...
+        public virtual DbSet<VUserTrack> VUserTracks { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. You can avoid scaffolding the connection string by using the Name= syntax to read it from configuration - see https://go.microsoft.com/fwlink/?linkid=2131148. For more guidance on storing connection strings, see http://go.microsoft.com/fwlink/?LinkId=723263.
+                optionsBuilder.UseSqlServer("Server=(localdb)\\mssqllocaldb;Database=DIMSCore;Trusted_Connection=True;");
+            }
+        }
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs:38]: The solution already has configuration of conn string in DAL assembly so you can remove this override

File: DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj [hunk 4]
```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.9" />
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj:12]: Why do you need this change?

File: DIMS-Core/DIMS-Core.Identity/DIMS-Core.Identity.csproj [hunk 5]
```diff
@@ -23,7 +23,7 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="5.0.1" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.1" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.9" />
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.Identity/DIMS-Core.Identity.csproj:26]: ??

File: DIMS-Core/DIMS-Core/DIMS-Core.csproj [hunk 6]
```diff
@@ -34,6 +34,7 @@
   <ItemGroup>
     <PackageReference Include="BuildBundlerMinifier" Version="3.2.449" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="5.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.9" />
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core/DIMS-Core.csproj:37]: ?

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj (hunk 4 above)
Comment (kuzmiich) [reply DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj:12]: why not?

File: DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs (hunk 3 above)
Comment (kuzmiich) [reply DIMS-Core/DIMS-Core.DataAccessLayer/Models/DIMSCoreContext.cs:38]: fixed

File: DIMS-Core/DIMS-Core.BusinessLayer/Converters/DirectionConverter.cs (hunk 1 above)
Comment (kuzmiich) [reply DIMS-Core/DIMS-Core.BusinessLayer/Converters/DirectionConverter.cs:27]: fixed

File: DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj (hunk 2 above)
Comment (kuzmiich) [reply DIMS-Core/DIMS-Core.DataAccessLayer/DIMS-Core.DataAccessLayer.csproj:31]: fixed

**Review by aces47 — APPROVED** [review]
Good work!

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**